README.md

```diff
@@ -0,0 +1,4 @@
+# Cromwell
+Cromwell is a replacement firmware for the original Xbox . It can boot the Linux operating system and practically convert the Xbox into a full PC.
+
+It is not able to load Xbox games (either original or otherwise). This is because the original Microsoft firmware image contains the kernel of the Xbox operating system which allows Xbox games to run and without the Microsoft image official Xbox software will no longer function.
```

Comment:
```suggestion
It is not able to load Xbox games (either original or otherwise), because the original Microsoft firmware image contains the kernel of the Xbox operating system which is responsible for running Xbox games and without it official Xbox software will no longer function.
```